main.tf

```diff
@@ -168,27 +171,30 @@ resource "null_resource" "icp-config" {
   provisioner "remote-exec" {
     inline = [
       "/tmp/icp-bootmaster-scripts/copy_cluster_skel.sh ${var.icp-inception == "" ? "" : " -v ${var.icp-inception}"}",
```

Comment:
I guess we can also update `copy_cluster_skel.sh` to take the now standardized `-i <icp-inception>` and pass the `${local.script_options}` for consistency

Reply:
This was an oversight. I already updated the `copy_cluster_skel.sh` to get the standardized parameters.